packages/flutter_tools/lib/src/commands/init.dart

```diff
@@ -201,14 +201,27 @@ void main() {
   runApp(
     new MaterialApp(
       title: "Flutter Demo",
-      routes: {
+      routes: <String, RouteBuilder>{
         '/': (RouteArguments args) => new FlutterDemo()
       }
     )
   );
 }
 
-class FlutterDemo extends StatelessComponent {
+class FlutterDemo extends StatefulComponent {
+  @override
```

Comment:
To make it useful, we'd have to use it everywhere. To use it everywhere, we need the tools to complain when it's missing. They don't complain when it's missing.

Once we can do analyzer plugins that's one of the things I may add support for.